includes/views/upgrade-table.php

```diff
@@ -38,21 +38,68 @@
 	<tr class="upgrade-links">
 		<td class="first" align="left">&nbsp;</td>
 		<?php
-		printf(
-			'<td><a href="%s" target="_blank">%s</a></td>', 
-			'https://wpovernight.com/downloads/woocommerce-pdf-invoices-packing-slips-professional/', 
-			__( 'Upgrade now', 'woocommerce-pdf-invoices-packing-slips' ) 
-		);
-		printf(
-			'<td><a href="%s" target="_blank">%s</a></td>', 
-			'https://wpovernight.com/downloads/woocommerce-pdf-invoices-packing-slips-premium-templates/', 
-			__( 'Upgrade now', 'woocommerce-pdf-invoices-packing-slips' ) 
-		);
-		printf(
-			'<td><a href="%s" target="_blank">%s</a></td>', 
-			'https://wpovernight.com/downloads/woocommerce-pdf-invoices-packing-slips-bundle/', 
-			__( 'Upgrade now', 'woocommerce-pdf-invoices-packing-slips' ) 
-		);
+			$extensions_disabled = [];
+			$extensions_enabled  = [];
+			$extension_columns   = [];
+			
+			// pro & templates columns
+			foreach ( $extension_license_infos as $extension => $info ) {
+				// enabled
+				if ( $info['enabled'] ) {
+					$extensions_enabled[] = $extension;
+					
+					$title = __( 'Currently installed', 'woocommerce-pdf-invoices-packing-slips' );
+					if ( empty( $info['status'] ) || $info['status'] != 'valid' ) {
+						$subtitle = sprintf(
+							/* translators: learn more link */
+							__( 'License not yet activated: %s', 'woocommerce-pdf-invoices-packing-slips' ),
+							'<a href="https://docs.wpovernight.com/general/installing-wp-overnight-plugins/" target="_blank">'.__( 'Learn more', 'woocommerce-pdf-invoices-packing-slips' ).'</a>'
+						);
+					} else {
+						$subtitle = __( 'License activated!', 'woocommerce-pdf-invoices-packing-slips' );
+					}
```

Comment:
As license info is secondary on this tab I don't think we need to display this. Only when the license is not activated we can give a little nudge for the user to activate it.